### PR TITLE
Feature/database info

### PR DESCRIPTION
**Pull request by joverthegrey**
@M52

### CODE REVIEW

File: src/class-database-size-plugin.php [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+    function __construct() {
+        add_action( 'admin_menu', [ $this, 'admin_menu_init' ]);
+        add_action( 'admin_bar_menu', [ $this, 'admin_bar_menu' ], 90);
+    }
+
+    function admin_menu_init() {
+        add_submenu_page(
+            'warpdrive_dashboard',
+            'Database information',
+            'Database information',
```
Comment (M52) [inline src/class-database-size-plugin.php:21]: Duplicate?

File: src/class-database-size-plugin.php [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+        $databaseinfo = Database::get_wp_database_size();
+        $errormsg = '';
+        $databasename = '';
+        $databasesize = 0;
+        if (
+                !array_key_exists('database', $databaseinfo) ||
+                !array_key_exists('size', $databaseinfo) ||
+                empty($tables)
+        ) {
+            $errormsg = "Could not retrieve database information, please contact support";
```
Comment (M52) [inline src/class-database-size-plugin.php:56]: Nice to have: actual email address here.

File: src/class-database.php [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+
+/**
+ * Class Database
+ * @package Savvii
+ */
+class Database {
+    public static function get_wp_table_sizes( $limit = -1 ) {
+        global $wpdb;
+
+        //check_admin_referer( 'warpdrive_viewdatabasesize' );
```
Comment (M52) [inline src/class-database.php:13]: Commented line can be removed.

**Review by M52 — DISMISSED** [review]
Can you check the possible duplicate "Database information" in class-database-size-plugin.php?

File: src/class-database-size-plugin.php (hunk 2 above)
Comment (joverthegrey) [reply src/class-database-size-plugin.php:56]: I'll add it

File: src/class-database-size-plugin.php (hunk 1 above)
Comment (joverthegrey) [reply src/class-database-size-plugin.php:21]: This is needed for the menu tree

File: src/class-database.php (hunk 3 above)
Comment (joverthegrey) [reply src/class-database.php]: 👍

**Pull request merged**